src/expr/symbol_table.cpp

```diff
@@ -73,7 +76,23 @@ class SymbolTable::Implementation {
   void pushScope() throw();
   size_t getLevel() const throw();
   void reset();
-
+  //------------------------ operator overloading
```

Comment:
For these functions, just flag that they are the implementation versions of the functions defined in the header so you don't have to repeat the method comments.

Reply:
Done